### PR TITLE
[Embeds] stop adding tracking params to non-bsky.app links

### PR DESCRIPTION
**Pull request by mozzius**
The `ref_src` and `ref_url` params were being added to the embed cards within the embed, which is undesirable. This adds a `disableTracking` prop to the link component to try and avoid that

### CODE REVIEW

File: bskyembed/.eslintrc [hunk 1]
```diff
@@ -15,6 +15,6 @@
   "parserOptions": {
     "sourceType": "module",
     "ecmaVersion": "latest",
-    "project": "./tsconfig.json"
+    "project": "./bskyembed/tsconfig.json"
```
Comment (gaearon) [inline bskyembed/.eslintrc:18]: ?

File: bskyembed/src/components/post.tsx [hunk 2]
```diff
@@ -123,7 +123,8 @@ function PostContent({record}: {record: AppBskyFeedPost.Record | null}) {
         <Link
           key={counter}
           href={segment.link.uri}
-          className="text-blue-400 hover:underline">
+          className="text-blue-400 hover:underline"
+          disableTracking={!segment.link.uri.startsWith('https://bsky.app')}>
```
Comment (gaearon) [inline bskyembed/src/components/post.tsx:127]: is `segment.link.uri` guaranteed to exist?

**Review by gaearon — APPROVED** [review]
lgtm

File: bskyembed/.eslintrc (hunk 1 above)
Comment (mozzius) [reply bskyembed/.eslintrc:18]: Eslint wouldn't let me merge unless I did this - for some reason it looks at this path from the perspective of the repo root? Very odd, I don't know if there's something else funny going on

File: bskyembed/src/components/post.tsx (hunk 2 above)
Comment (mozzius) [reply bskyembed/src/components/post.tsx:135]: Good shout, updated to use `validateLink` which is what the main app's richtext does

**Pull request merged**